### PR TITLE
new package: poppler-glib at 0.82.0

### PR DESCRIPTION
**Pull request by tristelune1**
## Description of package

Dependency needed for zathura the pdf viewer. 

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/poppler-glib/depends [hunk 1]
```diff
@@ -0,0 +1 @@
+libjpeg-turbo
```
Comment (dylanaraps) [inline community/poppler-glib/depends]: Missing `cmake make`.

Comment (dylanaraps) [reply community/poppler-glib/depends]: Missing `glib`.

Comment (dylanaraps) [reply community/poppler-glib/depends]: Missing `zlib`.

**Comment by tristelune1** [comment]
I did the following changes:

- updated to latest version: 0.82.0
- flags of build updated
- dependencies updated.

**Comment by dylanaraps** [comment]
Thanks :D

**Pull request merged**